ui/src/actions/types.js

```diff
@@ -1,6 +1,10 @@
 export const REVERSE_SORT = 'REVERSE_SORT';
 export const GET_APPS = 'GET_APPS';
```

Comment:
@craicoverflow am I right in saying we don't use either GET_APPS or GET_APP_BY_ID anywhere?

Reply:
You are right.